NSObject-SafeExpectations.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name         = "NSObject-SafeExpectations"
-  s.version      = "0.0.3"
+  s.version      = "0.0.4-beta.1"
```

Comment:
I don't expect this project to have more changes, we probably could go directly to 0.0.4

Reply:
👍